interleavebench/ddls.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// schemaVariant is the type of schema we want to benchmark on. Tables are
+// either interleaved (Interleaved) or not (Normal).
+type schemaVariant string
+
+const (
+	Normal      schemaVariant = "normal"
+	Interleaved               = "interleaved"
+)
+
+type tableName string
+
+const (
+	Merchant           tableName = "merchant"
+	Product                      = "product"
+	Variant                      = "variant"
+	Store                        = "store"
+	ProductInterleaved           = "product (interleaved)"
+	VariantInterleaved           = "variant (interleaved)"
+	StoreInterleaved             = "store (interleaved)"
+)
+
+var interleaveName = map[tableName]tableName{
+	Product: ProductInterleaved,
+	Variant: VariantInterleaved,
+	Store:   StoreInterleaved,
+}
+
+// Map of all DDL statements that are run when loadSchema is invoked.
+var ddlStmts = map[tableName]string{
+	Merchant: `
+	create table merchant (
```

Comment:
Nit: we normally capitalize SQL keywords. `s/create table/CREATE TABLE/g`

Reply:
Good point, done.